core/src/main/res/values/strings.xml

```diff
@@ -86,8 +86,7 @@
   <string name="tab_closed">Tab closed</string>
   <string name="bookmark_added">Bookmark added</string>
   <string name="rate_dialog_title">Please Rate Us</string>
-  <string name="rate_dialog_msg_1">If you enjoy using</string>
-  <string name="rate_dialog_msg_2">, please take a moment to rate it. Thanks for your support!</string>
+  <string name="rate_dialog_msg">If you enjoy using Kiwix , please take a moment to rate it. Thanks for your support!</string>
```

Comment:
Breaking it half did a great job with custom apps as well i.e message1 + app name + message 2. Is it okay to show this message `If you enjoy using Kiwix, please` to custom apps as well?
cc @kelson42

Reply:
@gouri-panda yes, but where is the job/string about that? If this is in the custom repo?

Reply:
This is from core. It will work both on kiwix and custom apps simultaneously.

Reply:
@gouri-panda I don't understand. Can you please open an other ticket for that? We should merge and release.

Reply:
I understand now, this PR can not work like this indeed. We should have a string like `If you enjoy using %s , please take a moment to rate it. Thanks for your support!`. With `%s` (this is just an example, I don't know exactly what should be the system) been replaced at runtime dynamically by the app name. This is necessary for the custom apps which have a different name.